src/main/java/org/folio/rest/impl/FeeFinesAPI.java

```diff
@@ -42,6 +46,10 @@
     private static final String FEEFINE_ID_FIELD = "'id'";
     private static final String OKAPI_HEADER_TENANT = "x-okapi-tenant";
     private final Logger logger = LogManager.getLogger(FeeFinesAPI.class);
+    private final List<String> requiredFeeFineIdList = List.of("9523cb96-e752-40c2-89da-60f3961a488d",
```

Comment:
We call these fee/fine types automatic - check populate-feefines.sql
Also, should we make an enum for these values?

Reply:
so, you propose to create an enum for these ID values, right?

Reply:
Yes. It would be nice to have a single place in the code with this list, otherwise we'll be copy/pasting these values everywhere. You can make it similar to patron block conditions in mod-patron-blocks

Reply:
done